tensorflow/contrib/verbs/rdma_rendezvous_mgr.cc

```diff
@@ -58,20 +60,13 @@ void RdmaRemoteRendezvous::RecvFromRemoteAsync(
   // parse src_name and dst_name
   string src_name, dst_name, unused;
   if (!DeviceNameUtils::SplitDeviceName(parsed.src_device, &src_name,
+                                        &unused) ||
+      !DeviceNameUtils::SplitDeviceName(parsed.dst_device, &dst_name,
                                         &unused)) {
     s = errors::Internal("Could not parse src name.");
```

Comment:
Looks like the error message is misleading if it's the dst_name that won't parse.

Reply:
It's scr or dst name. Fixed in [d43d00b](https://github.com/tensorflow/tensorflow/pull/14290/commits/d43d00be13ff271eb8a2e6a14eb7ac01a51934ff).